### DISCUSSION TITLE
Dashboard button URL - open in current window

### DISCUSSION THREAD

**Discussion by rbreneman** [home-assistant/frontend#14564] [Dashboard]
Could an option be added to the button card when using the URL action, to open in current window rather than a new tab?

**Comment by Mariusmssj** [comment]
I am curious about this as well. I am trying to setup a dashboard with multiple views and use the url field to switch between the but some reason when I click on the button it just opens a new tab and on the mobile app it launches a web browser which seems odd to me.
